yt_dlp/extractor/mxplayer.py

```diff
@@ -14,7 +15,7 @@
 
 
 class MxplayerIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?mxplayer\.in/(?:show|movie)/(?:(?P<display_id>[-/a-z0-9]+)-)?(?P<id>[a-z0-9]+)'
+    _VALID_URL = r'(?:https?://)(?:www\.)?mxplayer\.in/(?:show/[-\w]+/[-\w]+|movie)/(?P<display_id>[-\w]+)-(?P<id>\w+)'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?mxplayer\.in/(?:movie|show/[-\w]+/[-\w]+)/(?P<display_id>[-\w]+)-(?P<id>\w+)'
```
may be easier to read. Or even better, split it into multiple lines